sisl/geom/category/_coord.py

```diff
@@ -232,3 +233,55 @@ def __eq__(self, other):
                         return False
                 return True
         return False
+
+# The following code is just to make the use of coordinate categories more convenient
+# We will create one individual category for each direction. In this way, one can access
+# them directly in the keyword category builder. E.g.: `AtomCategory(x=(1,2))` creates `AtomXYZ(x=(1,2))`
+# without needing to nest "x", as in `AtomCategory(xyz={"x": (1,2)})`.
+
+
+class CoordMeta(CategoryMeta):
```

Comment:
To conform to the class name, name this `AtomXYZMeta`, I guess this meta is only applicable to this class, so explicitness is good! :+1:

Reply:
I changed the name, but this could be applicable to neighbours for example, right?

Reply:
Well, just the concept, probably not exactly this meta.